components/atoms/badge/components/badge.js

```diff
@@ -53,5 +53,5 @@ Badge.propTypes = {
   content: string.isRequired,
   to: string,
   context: oneOf(Object.values(CONTEXT)),
-  icon: object
+  icon: string
```

Comment:
@majid-amiri You will want a prefix here then also, as icons for fontawesome can come in several packages: see the icon compinent: icon={[prefix, icon]}

Reply:
Done.